x-pack/plugins/cases/public/components/filter_popover/index.tsx

```diff
@@ -21,7 +21,7 @@ interface FilterPopoverProps {
   buttonLabel: string;
   onSelectedOptionsChanged: Dispatch<SetStateAction<string[]>>;
   options: string[];
-  optionsEmptyLabel: string;
+  optionsEmptyLabel?: string;
```

Comment:
Why the need to make it optional?

Reply:
The filter for solution type will always show up if the user has 2 or more solution types available to them (observability & securitySolution). Currently though, the FilterPopover expects a prop `optionsEmptyLabel` that it shows when whatever youre filtering by does not have any options:

For eg if youre filtering by tags, but none of the cases have tags then there isnt any filter option to select.

In this use case though, this cant happen. Either a user has 2 or more options to filter by, and we therefore show the Filter Select, or they don't and the entire Filter Select is hidden.

The alternative fix was to pass in the prop `OptionsEmptyLabel` with a string that would never actually be rendered to the page.